workflow/activity_options.go

```diff
@@ -35,47 +35,77 @@ type LocalActivityOptions = internal.LocalActivityOptions
 // RetryPolicy specify how to retry activity if error happens.
 type RetryPolicy = internal.RetryPolicy
 
-// WithActivityOptions adds all options to the copy of the context.
+// WithActivityOptions makes a copy of the context and adds the
+// passed in options to the context. If an activity options exists,
```

Comment:
I think it worth to emphasize on what will happen if input options has default properties.